.github/workflows/ai-pr-review.yml

```
@@ -6,7 +6,7 @@ on:
   workflow_dispatch:
     inputs:
       pr_number:
-        description: 'Pull request number to review'
+        description: "Pull request number to review"
```

Isn't that a string automatically?
```suggestion
        description: Pull request number to review
```

---

funnily enough I didn't change that on purpose. I guess prettier/biome went ahead and formatted it for me